content/java/concepts/calendar/terms/clear.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: '.clear()'
+Description: 'A method designed to reset or clear specific fields of the calendar instance.'
+Subjects: 
+    - 'Computer Science'
+Tags:
+    - 'Methods'
+    - 'Calendar'
+CatalogContent:
+    - 'learn-java'
+    - 'paths/computer-science'
+---
+
+The method **`clear()`** in the [`Calendar`](https://www.codecademy.com/resources/docs/java/calendar) class is used to reset or clear specific fields of the calendar instance to their default values. It allows you to reset individual fields, such as `YEAR`, `MONTH`, `DAY_OF_MONTH`, `HOUR`, etc., to their initial state.
+
+## Syntax
+
+```pseudo
+void clear()
+void clear(int field)
+```
+
+> **Note:** Without any parameters, the `clear` method clears all fields of the Calendar instance, thereby resetting it to its initial state. The `clear(int field)` method allows you to clear a specific field by providing the corresponding field constant from the Calendar class.
+
+## Example
+
+In this example, the `CalendarClearExample` class demonstrates the use of `clear()` and `clear(int field)` methods. The first part clears all fields of the Calendar instance, while the second part clears the YEAR field, effectively setting it to its default value (which is 1970).
+
+```java
+import java.util.Calendar;
+
+public class CalendarClearExample {
+    public static void main(String[] args) {
```

Comment:
Docs does not use tabs, please update the examples so that indentations are 2 spaces